packages/components/.swcrc

```diff
@@ -25,5 +28,8 @@
     "strictMode": true,
     "lazy": false,
     "noInterop": false
+  },
+  "env": {
+    "targets": "> 0.25%, not dead"
```

Comment:
why this?

Reply:
I mean, I know what it does, the question is why do we need it?

Reply:
that's the default for all (production) react apps (we are using the same target in auto-tune and I can bet other projects). We don't need to target old browsers

Reply:
more info: https://github.com/browserslist/browserslist#best-practices

Reply:
but do we really need this?

Reply:
it's building all the components so this code will be interpreted by browsers, so if there is a change to avoid some quirks this config should do it. But I can remove it, It will work in the same way